tests/unit/amo/components/TestCollection.js

```diff
@@ -703,9 +694,7 @@ describe(__filename, () => {
     const wrapper = renderComponent({ store, _config: fakeConfig });
 
     const editLink = wrapper.find('.Collection-edit-link').find(Button);
-    expect(editLink).toHaveLength(1);
-    expect(editLink).toHaveProp('href',
-      `/collections/${defaultUser}/${defaultCollectionDetail.slug}/edit/`);
+    expect(editLink).toHaveLength(0);
```

Comment:
You have effectively removed this test, but that's not what we want. We just need to change the test so that it accesses the page with a user who is also the owner of the collection.